mobile-save-for-later-user-deletion/riff-raff.yaml

```diff
@@ -9,3 +9,8 @@ deployments:
       functionNames: [mobile-save-for-later-user-deletion-]
       fileName: mobile-save-for-later-user-deletion.jar
       prefixStack: false
+  mobile-save-for-later-user-deletion-cfn:
+    type: cloud-formation
+    app: mobile-save-for-later-user-deletion
+    parameters:
+      templatePath: cfn.yaml
```

Comment:
Did you try deploying this to `CODE` yet? I'm not 100% sure but I think we might need to specify the template path as `conf/cfn.yaml`?

Reply:
it worked without adding the `conf` part 🤷‍♀️ shall I add it just incase?

Reply:
This part of Riff-Raff has always confused me 🤦 

If it works then there's no need to change it!